app/domain/api/request.rb

```diff
@@ -5,27 +5,39 @@ class Api::Request
 
   include ActiveModel::Validations
 
-  attr_reader :metric, :from, :to, :base_path
+  attr_reader :metric, :from, :to, :base_path, :metrics
 
-  validates :metric, presence: true, inclusion: { in: Metric.find_all.map(&:name) }
   validates :from, presence: true, format: { with: DATE_REGEX, message: "Dates should use the format YYYY-MM-DD" }
   validates :to, presence: true, format: { with: DATE_REGEX, message: "Dates should use the format YYYY-MM-DD" }
   validates :base_path, presence: true
   validate :from_before_to, if: :have_a_date_range?
+  validate :verify_metrics, if: :have_multiple_metrics
```

Comment:
Why do we check if they have `multiple_metrics`? In theory we should only have at least one.